src/screens/GroupInfo/elements/ModalAction.js

```diff
@@ -21,7 +30,10 @@ const ModalAction = (props) => {
           testID="pressMessage"
           onPress={() => onPress('message', selectedUser)}
           style={styles.buttonStyle}>
-          <Text style={styles.textButton}>Message {name}</Text>
+          <View style={{flexDirection: 'row'}}>
+            <Text style={styles.textButton}>Message {name}</Text>
+            <Loading visible={isLoadingInitChat} />
+          </View>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `Loading` component is conditionally rendered based on the `isLoadingInitChat` prop. This is a good way to provide feedback to the user about the loading state. However, make sure that the `Loading` component does not disrupt the layout when it appears and disappears.

```diff
- <Text style={styles.textButton}>Message {name}</Text>
+ <View style={{flexDirection: 'row'}}>
+   <Text style={styles.textButton}>Message {name}</Text>
+   <Loading visible={isLoadingInitChat} />
+ </View>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->